examples/color/named_colors.py

```diff
@@ -16,7 +16,8 @@
 from matplotlib import colors as mcolors
 
 
-colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS)
+colors = dict(mcolors.BASE_COLORS, **mcolors.CSS4_COLORS,
+                **mcolors.SOLARIZED_COLORS)
```

Comment:
Please align the continued line with the inside I’d the parentheses, I.e. remove two spaces.